spec/features/consumer/shopping/checkout_paypal_spec.rb

```diff
@@ -44,7 +44,8 @@
   describe "as a guest" do
     it "fails with an error message" do
       visit checkout_path
-      complete_the_form
+      checkout_as_guest
+      fill_out_form(free_shipping.name, paypal.name, false)
```

Comment:
I don't find this helper very clear, especially the last parameter doesn't have a name. Maybe use a keyword argument here?
https://thoughtbot.com/blog/ruby-2-keyword-arguments

Reply:
that's what Pau suggested, you probably saw it on the last commit.